client/src/commonMain/kotlin/com/algolia/search/model/filter/Filter.kt

```diff
@@ -32,20 +32,25 @@ public sealed class Filter {
 
         public sealed class Value {
 
+            /**
+             * Raw value.
+             */
+            public abstract val raw: Any
+
             /**
              * Filter on a [kotlin.String] value.
              */
-            public data class String(val raw: kotlin.String) : Value()
+            public data class String(override val raw: kotlin.String) : Value()
```

Comment:
What is this change for?

Reply:
It's to have access to the actual raw value of the `Facet.Value`, without caring which implementation it is (`String`, `Boolean` or `Numeric`)